husky_directory/templates/results.html

```diff
@@ -0,0 +1,17 @@
+
+
+{% for key,value in search_result.items() %}
+    {% if (value['people']|length) %}
+        <p>
+            <h4>{{key}}</h4>
+                {% for data in value['people'] %} {# people is an array of objects #}
```

Comment:
NB: It might be good to just link directly to the Person model in your comment so that it's easy to reference. Not something you need to do right now.